docusaurus/docs/testing/extensions-test.md

```diff
@@ -0,0 +1,32 @@
+# Extensions Tests
+
+The Extensions framework is tested by the GitHub action `check-plugins.yaml`. This runs the script `./shell/scripts/test-plugins-build.sh`
+which runs the actual tests.
+
+The tests check the following:
+
+- Ensures that Verdaccio is installed and running locally (Verdaccio provides a local NPM registry that the tests can use to publish packages without having to publish to npm)
+- Verifies the build of the following, by building and then publishing them to the local registry
+  - Shell
+  - Yarn Creator packages
+  - Rancher components package
+- Verifies that a new app can be created with the `@rancher/app` creator and built (test case of a new extension in a new repository)
+- Verifies that an extension can be added with the `@rancher/pkg` creator to the new app and that the dashboard build passes with the new extension present
```

Comment:
While creating the pkg it seems like there's some kind of configuration although I see nothing obvious:

```
  Output directory: /home/runner/work/dashboard/dashboard/dist-pkg/test-pkg-0.1.0
~/work/dashboard/dashboard/pkg/test-pkg ~/work/dashboard/dashboard
```

It [generates conflicts](https://github.com/rancher/dashboard/actions/runs/4723080482/jobs/8378536519?pr=8654#step:4:830) on version changes which are probably related to some libraries defined in a `package.json` somewhere, I guess?

Reply:
Ok I just found out on my own, the paths are:
- `shell/creators/app`
- `shell/creators/pkg`
- `shell/creators/update`

Could we please add this to the documentation, especially if we have to make changes or upgrades?